include/ofxColour.h

```diff
@@ -0,0 +1,175 @@
+#ifndef _ofxColour_h_
+#define _ofxColour_h_
+
+// Copyright OpenFX and contributors to the OpenFX project.
+// SPDX-License-Identifier: BSD-3-Clause
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @file ofxColour.h
+Contains the API for colourspace data exchange.
+*/
+
+/** @brief What style of colour management does the host or plug-in offer?
+
+   - Type - string X 1
+   - Property Set - host descriptor (read only), plugin descriptor (read/write)
+   - Default - kOfxImageEffectPropColourManagementStyleNone
+   - Valid Values - This must be one of
+     - ::kOfxImageEffectPropColourManagementNone - no colour management
+     - ::kOfxImageEffectPropColourManagementNative - colourspaces from the OCIO built-in ACES Studio config are available (see ofxColourspaceList.h)
```

Comment:
I think this needs a little clarification.
* Are only colourspaces in `ofxColourspaceList.h` permitted, or can a host add its own names, on the assumption that some plugin with insider knowledge can make use of them?
* Does a host/plugin have to support *every* colourspace listed in that file? Clearly not, so I think this comment needs to be clear that these are the set of spaces, some or all of which a host is expected to offer support.
* Are any spaces more required than others? Is `data` required? Is `raw` required?